Description: Fix FTBFS with FFmpeg 4.0
 The EMU_EDGE codec flag was deprecated in FFmpeg 2.2 and has done nothing since
 then, so it should be fine to remove.
Author: James Cowgill <jcowgill@debian.org>
Bug-Debian: https://bugs.debian.org/888358
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/media_object.cpp
+++ b/src/media_object.cpp
@@ -898,15 +898,6 @@ void media_object::open(const std::strin
             // Activate multithreaded decoding. This must be done before opening the codec; see
             // http://lists.gnu.org/archive/html/bino-list/2011-08/msg00019.html
             codec_ctx->thread_count = video_decoding_threads();
-            // Set CODEC_FLAG_EMU_EDGE in the same situations in which ffplay sets it.
-            // I don't know what exactly this does, but it is necessary to fix the problem
-            // described in this thread: http://lists.nongnu.org/archive/html/bino-list/2012-02/msg00039.html
-            int lowres = 0;
-#ifdef FF_API_LOWRES
-            lowres = codec_ctx->lowres;
-#endif
-            if (lowres || (codec && (codec->capabilities & CODEC_CAP_DR1)))
-                codec_ctx->flags |= CODEC_FLAG_EMU_EDGE;
         }
         // Find and open the codec. AV_CODEC_ID_TEXT is a special case: it has no decoder since it is unencoded raw data.
         if (codec_ctx->codec_id != AV_CODEC_ID_TEXT && (!codec || (e = avcodec_open2(codec_ctx, codec, NULL)) < 0))
